docs/src/main/asciidoc/security-openid-connect-multitenancy.adoc

```diff
@@ -268,14 +275,19 @@ public class CustomTenantResolver implements TenantResolver {
 }
 ----
 
-You can define multiple tenants in your configuration file, just make sure they have a unique alias so that you can map them properly when resolving a tenant from your `TenantResolver` implementation.
+You can define multiple tenants in your configuration file.
+To map them correctly when resolving a tenant from your `TenantResolver` implementation, ensure each has a unique alias.
 
-However, using a static tenant resolution (configuring tenants in `application.properties` and resolving them with `TenantResolver`) prevents testing the endpoint with `Dev Services for Keycloak` since `Dev Services for Keycloak` has no knowledge of how the requests will be mapped to individual tenants and can not dynamically provide tenant-specific `quarkus.oidc.<tenant-id>.auth-server-url` values and therefore using `%prod` prefixes with the tenant-specific URLs in `application.properties` will not work in test or dev mode.
+However, using a static tenant resolution method, such as configuring tenants in `application.properties` and resolving them with `TenantResolver`, presents limitations.
+This approach is not suitable for testing the endpoint with Dev Services for Keycloak.
+The reason is that Dev Services for Keycloak does not understand how requests are mapped to individual tenants.
```

Comment:
```suggestion
... because it does not understand how requests are mapped to individual tenants.
```
and adding it back to the prev sentence will work fine IMHO

Reply:
Updated. Thanks.